### PR TITLE
Fix for ts and prop types

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/components/ChannelList/hooks/useChannelDeletedListener.js [hunk 1]
```diff
@@ -30,7 +39,7 @@ export const useChannelDeletedListener = (setChannels, customHandler) => {
     });
 
     return () => {
-      client.off('channel.deleted');
+      client.off('channel.deleted', () => null);
```
Comment (vishalnarkhede) [inline src/components/ChannelList/hooks/useChannelDeletedListener.js:42]: Please change it to:

```js
    client.on('channel.deleted', handleEvent);

    return () => {
      client.off('channel.deleted', handleEvent);
    };
```

Comment (vishalnarkhede) [reply src/components/ChannelList/hooks/useChannelDeletedListener.js:42]: And same for other hooks as well :)

Comment (nhannah) [reply src/components/ChannelList/hooks/useChannelDeletedListener.js]: Changed for all.

**Comment by vishalnarkhede** [comment]
LGTM!!

**Pull request merged**